src/services/paras/ParasService.ts

```diff
@@ -223,21 +223,47 @@ export class ParasService extends AbstractService {
 			phase: IOption<AuctionPhase>,
 			winning;
 		if (auctionInfoOpt.isSome) {
+			/**
+			 * If `AuctionInfo:::<T>:get()` is `Some`, it returns a tuple where the first item is the
+			 * lease period index that the first of the four contiguous lease periods
+			 * an auction is for. The second is the block number when the auction will
```

Comment:
```suggestion
			 * of an auction is for. The second is the block number when the auction will
```

Reply:
Just looked at this again and I'm confused at why I suggested adding `of`; clearly my brain wasn't working :D